integration_tests/tests/db.rs

```diff
@@ -2,9 +2,9 @@ use diesel::prelude::*;
 use diesel::r2d2;
 
 pub type DieselPool = r2d2::Pool<r2d2::ConnectionManager<PgConnection>>;
-pub type PoolBuilder = r2d2::Builder<r2d2::ConnectionManager<PgConnection>>;
+pub type PoolBuilder = swirl::db::R2d2Builder;
 
-pub fn pool_builder() -> PoolBuilder {
+pub fn pool_builder() -> r2d2::Builder<r2d2::ConnectionManager<PgConnection>> {
```

Comment:
Why is this no longer the type alias from above?

Reply:
Most places using that type alias needed to switch to the new value it has. This was the only place which is continuing to return an `r2d2::Builder`. Since the new value of the type alias is used more frequently, I felt it was better to change the return type of this function than everywhere else using the alias